targets/arm32/cm3/atmel/sam3x/sam3x8e/flashscripts/arduino_due.cmake

```diff
@@ -9,17 +9,17 @@ set(DEVICE_LOCATION
 )
 
 if(bossac)
-    if(WIN32)
+    if(MSYS OR MINGW)
```

Comment:
What if i use other build systems like ninja on windows?
what was wrong with checking for win32?